### DISCUSSION TITLE
What's ggml-large-v3-q5_0.bin?

### DISCUSSION THREAD

**Discussion by playgithub** [ggml-org/whisper.cpp#2316] [Q&A]
ggml-large-v3-q5_0.bin is about 1.08GB,
ggml-large-v3.bin is about 3.1GB.
What's the difference?

**Comment by itsthisjustin** [comment]
The one with the Q is a quantized model

**Reply by playgithub** [reply]
What's a quantized model?

**Reply by jensdraht1999** [reply]
It's a model, where the data is "compressed" by removing redundant data. Since the data is removed, it is using less VRAM/RAM, takes less space on your device and works faster, because it's smaller. In the case of large3-v3, it is 3 times smaller.
